apps/torus-worker/src/index.ts

```diff
@@ -16,21 +17,45 @@ export const env = parseEnvOrExit(
   }),
 )(process.env);
 
+const log = createLogger({ name: "index" });
+
+/**
+ * Sets up and creates a connection to the Torus blockchain node
+ *
+ * @returns A connected API Promise instance
```

Comment:
"Sets up and creates a connection to the Torus blockchain node"

`: Promise<ApiPromise>`

"`@returns A connected API Promise instance`"

REDUNDANT